documentation/modules/proc-configuring-kafka-rack-awareness.adoc

```diff
@@ -10,7 +10,7 @@ The `rack` object has one mandatory field named `topologyKey`.
 This key needs to match one of the labels assigned to the Kubernetes cluster nodes.
 The label is used by Kubernetes when scheduling the Kafka broker pods to nodes.
 If the Kubernetes cluster is running on a cloud provider platform, that label should represent the availability zone where the node is running.
-Usually, the nodes are labeled with `failure-domain.beta.kubernetes.io/zone` that can be easily used as the `topologyKey` value.
+Usually, the nodes are labeled with `topology.kubernetes.io/zone` label (or `failure-domain.beta.kubernetes.io/zone` on older Kubernetes versions) that can be easily used as the `topologyKey` value.
```

Comment:
maybe we can specify what is the latest Kubernetes version still supporting the old label?

Reply:
AFAIK it is only deperecated. Not yet removed.

Reply:
Yes you are right, as I read in this page [1] "Starting in v1.17, this label is deprecated in favor of topology.kubernetes.io/zone". I would add this note here.
[1] https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/

Reply:
Yes you are right, as I read here [1] "Starting in v1.17, this label is deprecated in favor of topology.kubernetes.io/zone". I would add the same note in our documentation.
[1] https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/

Reply:
I added there a link to the Kubernetes page - it might have the version where they remove it in the future. Is that ok?

Reply:
That's fine thanks.